src/zotonic_sup.erl

```diff
@@ -248,3 +258,11 @@ get_extensions() ->
           permanent, 5000, worker, dynamic}
      end
      || F <- Files].
+
+%% @doc Let sites return their own keys and certificates.
+sni_fun(ServerName) ->
+    case z_sites_dispatcher:get_host_for_domain(ServerName) of
+        undefined -> undefined;
+        {ok, Host} -> 
+            z_notifier:first(#ssl_options{server_name=ServerName}, z_context:new(Host))
```

Comment:
This is a new notification that should be documented. Can you add it to the SSL module’s RST?